php/libraries/Database.class.inc

```diff
@@ -641,8 +641,7 @@ class Database
      */
     function selectOne($query)
     {
-        $result = null;
-        $this->select($query, $result);
+        $result = $this->pselect($query);
```

Comment:
this require a second argument

Reply:
yes, I guess that's one of the error types which are still suppressed by phan.. I'll fix it.